modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -43,48 +45,198 @@ The properties for broker-wide throughput quota balancing are configured at the
+To set a throughput quota for a single client, use the `rpk cluster quotas alter` command. 
 
-The values of both `target_quota_byte_rate` and `target_fetch_quota_byte_rate` are throughput rate limits within a shard, in bytes per second.
+[,bash]
+----
+rpk cluster quotas alter --add consumer_byte_rate=200000 --name client-id=consumer-1
+----
++
+[,bash,role=no-copy]
+----
+ENTITY                STATUS
+client-id=consumer-1  OK
+----
 
 === Group of clients throughput limit
 
-To set throughput quota for a group of clients, configure the cluster properties xref:reference:cluster-properties.adoc#kafka_client_group_byte_rate_quota[`kafka_client_group_byte_rate_quota`] and xref:reference:cluster-properties.adoc#kafka_client_group_fetch_byte_rate_quota[`kafka_client_group_fetch_byte_rate_quota`].
+Alternatively, you can view or configure throughput quotas for a group of clients based on a match on client ID prefix. The following example sets the `consumer_byte_rate` quota to client IDs prefixed with `consumer-`:
+
+[,bash]
+----
+rpk cluster quotas alter --add consumer_byte_rate=200000 --name client-id-prefix=consumer-
+----
+
+NOTE: A client group specified with `client-id-prefix` is not the equivalent of a Kafka consumer group. It is used only to match requests based on the `client_id` prefix. The `client_id` field is typically a configurable parameter when you create a client with Kafka libraries.
+
+=== Default client throughput limit
+
+You can apply default throughput limits to clients. Redpanda applies the default limits if no quotas are configured for a specific `client_id` or prefix.
+
+To specify a produce quota of 1GBps through the Kafka API, which means a 1GBps limit across all produce requests sent to a single Redpanda broker: 
+
+[,bash]
+----
+rpk cluster quotas alter --default client-id --add producer_byte_rate=1000000000
+----
+
+=== Bulk manage client throughput limits
 
-* The `kafka_client_group_byte_rate_quota` property applies to producer clients. It sets a maximum throughput quota of traffic sent to Redpanda from each producer in the group named by the property.
-* The `kafka_client_group_fetch_byte_rate_quota` property applies to consumer clients. It sets a maximum throughput quota of traffic fetched from Redpanda by each consumer in the group named by the property.
+To more easily manage multiple quotas, you can use the `cluster quotas describe` and `cluster quotas import` commands to do a bulk export and update.
 
-Both `kafka_client_group_byte_rate_quota` and `kafka_client_group_fetch_byte_rate_quota` have the configuration value fields:
+For example, to export all client quotas in JSON format:
 
-* `group_name`: a name for the group of clients.
-* `clients_prefix`: a client belonging to the group has this prefix in its `client_id` property.
-* `quota`: the maximum throughput rate of *each client in the group* in bytes per second.
+[,bash]
+----
```

Comment:
@r-vasquez does `cluster quotas describe --format json` output pure JSON and not, say, to a JSON file? If it's pure JSON, should we tell users to save the command output to a file so they can just provide the filepath as in the `import` example below? Or is it preferable to provide the `--source` argument as a string?

Reply:
`rpk cluster quotas import --from` accepts either a JSON, a YAML, or a file containing a JSON/YAML. 

`rpk cluster quotas describe --format json` returns the pure JSON.

So, I think it's fine to tell users to use that string, but we can mention that can be saved to files and used accordingly